common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -35,4 +33,4 @@
   % if msg:
       <span class="message">${HTML(msg)}</span>
   % endif
-</div>
+</form>
```

Comment:
@muhammad-ammar can you change this back to div and add make sure that the form element is in the parent template?  We only want one form element per "Check" button.